src/components/Article.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({
+  title,
+  date,
+  text,
+}) => (
+  <div>
+    <h1>
+      {title}
+    </h1>
```

Comment:
```suggestion
    <h1> {title} </h1>
```
если контент короткий, не всегда нужно переносить все на новую строку